app/assets/stylesheets/refills/_hero.scss

```diff
@@ -0,0 +1,64 @@
+.hero {  
+  ///////////////////////////////////////////////////////////////////////////////////
+  $base-border-radius: 3px !default;
+  $base-accent-color: #477DCA !default;
+  $large-screen: em(860) !default;
+  //////////////////////////////////////////////////////////////////////////////////
+
+  $hero-background-top: #7F99BE; 
+  $hero-background-bottom: #20392B;
+  $hero-color: red;
+  $gradient-angle: 10deg;
+  $hero-image: asset_path('hero-doge');
+
+  @include background(url($hero-image), linear-gradient($gradient-angle, $hero-background-bottom, $hero-background-top), no-repeat $hero-background-top scroll);
+  background-color: #324766;
```

Comment:
Color literals like `#324766` should only be used in variable declarations; they should be referred to via variable everywhere else.